apps/log/lib/log/model/log.ex

```diff
@@ -61,7 +61,7 @@ defmodule Helix.Log.Model.Log do
     %__MODULE__{}
     |> cast(params, @creation_fields)
     |> validate_required(@required_fields)
-    |> put_primary_key()
+    |> put_change(:log_id, PK.pk_for(__MODULE__))
```

Comment:
What should I do with [this](https://github.com/kress95/Helix/blob/c882643e8a8e41993eb80832283933786d192b6a/apps/log/lib/log/model/log.ex#L66-L79)?